plugins/source/aws/resources/services/s3/bucket_encryption_rules.go

```diff
@@ -22,9 +22,10 @@ func bucketEncryptionRules() *schema.Table {
 		Columns: []schema.Column{
 			client.DefaultAccountIDColumn(false),
 			{
-				Name:     "bucket_arn",
-				Type:     arrow.BinaryTypes.String,
-				Resolver: schema.ParentColumnResolver("arn"),
+				Name:       "bucket_arn",
+				Type:       arrow.BinaryTypes.String,
+				Resolver:   schema.ParentColumnResolver("arn"),
+				PrimaryKey: true,
```

Comment:
Thanks for the PR @smokentar, I'm not this is sufficient as wit the parent bucket as the only primary key it will allow saving a single rule per bucket.

Maybe the solution should be to remove this table and have the rules as an additional JSON column on the bucket table

Reply:
Thanks for the additional context @smokentar! I only looked at the AWS Go SDK struct of `ServerSideEncryptionConfiguration.Rules` and assumed that if it's an array, multiple items can be returned.

If there can only be a single rule (for now), this looks great 🚀